ArtOfIllusion/src/artofillusion/RotateViewTool.java

```diff
@@ -154,18 +160,18 @@ private void dragRotateTravelSpace(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown && !e.isShiftDown())
+		if (mouseButtonTwo(e) && controlDown)
 		{
 			view.tilting = true;
 			tilt(e, view, clickPoint);
 			return;
 		}
-		else if (controlDown && e.isShiftDown())
+		else if (mouseButtonTwo(e))
```

Comment:
```suggestion
		else if (controlDown && e.isShiftDown())
```
This is the temporary switch from spatial pan to rotate by pressing the control an shift keys simultaneously. 

The mouse button alone that took us to this part of the code is not the sufficient condition to activate the function.

Reply:
got it